src/xrGame/gsc_dsigned_ltx.cpp

```diff
@@ -75,10 +75,10 @@ bool gsc_dsigned_ltx_reader::load_and_verify(u8* buffer, u32 const size)
     *dsign_section = 0;
     xr_strcat(dsign_section, dsign_sect_size, ltx_date.c_str());
     u32 new_size = size - dsign_sect_size + ltx_date.size() + 1; // xr_strlen(buffer)
+    
```

Comment:
You removed redundant braces around one-line block (which is fine), but then inserted two empty lines. Vertical space is still wasted, but the diff is larger. I suggest you either undo this completely or not remove 2 new empty lines.

Reply:
I will remove empty lines here

Reply:
I don't see a problem if the diff will be larger for just two lines. But without them, code readability will suffer.

Reply:
Code was somehow readable with the braces, but now, without even a single empty space, it's more unhandy to read the code.